src/main/java/io/wgj/ctci/Stack.java

```diff
@@ -0,0 +1,21 @@
+package io.wgj.ctci;
+
+public class Stack {
+    int[] stack;
+    int position;
+
+    public Stack(){
+        stack = new int[10];
```

Comment:
Consider:
* Using generics for the stack type instead of just using integer.
* Making the maximum stack size passed in to the constructor (instead of hard-coded).

If the max-stack size *should* be hard-coded (not sure what your req's are for this), then I'd recommend making it a constant that can be referenced and used from the tests too.

Reply:
Fixed in cd6dc83cc0daf94827e214ba2a7c26f33c9aef00